Dockerfile

```diff
@@ -13,7 +13,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         python3 \
         runit \
         gnupg-agent \
-	socat \
+	    socat \
+	    libdpkg-perl \
```

Comment:
This indentation doesn't match, and I'm not sure `libdpkg-perl` is actually needed anywhere?